### PR TITLE
Initial branding updates

### PR DESCRIPTION
**Pull request by wpscholar**
Updated name of the main plugin file. Updated plugin headers to be Bluehost branded. Added upgrade handler. Updated version to 1.4.3 (can tag at any time later) and added upgrade routine for version 1.4.3. Removed all references to the `mm_brand` option.

### CODE REVIEW

**Comment by earnjam** [comment]
Should we be changing the `MM_` constant prefix, and the `mm_` function prefix too?

**Comment by wpscholar** [comment]
@earnjam We discussed in chat, but just to leave a record here as well: 

The goal is to make minimal changes so as to limit the possibility for breakage due to heavy refactoring. This PR changes the essential branding components that need to be handled during the plugin switch from Mojo to Bluehost. The remaining items will be refactored later.

File: bluehost.php [hunk 1]
```diff
@@ -1,33 +1,55 @@
...
+
 // Composer autoloader
 if ( version_compare( phpversion(), 5.3, '<' ) ) {
 	require dirname( __FILE__ ) . '/vendor/autoload_52.php';
 } else {
 	require dirname( __FILE__ ) . '/vendor/autoload.php';
 }
 
+// Handle any upgrade routines
+if ( is_admin() ) {
```
Comment (0aveRyan) [inline bluehost.php]: @wpscholar what do you think of making this `is_user_logged_in()`?

File: inc/menu.php [hunk 2]
```diff
@@ -9,24 +9,9 @@ function mm_main_menu() {
...
-		$menu_name = 'Bluehost';
-	}
-
-	if ( 'Bluehost_India' == $menu_name ) {
-		$menu_name = 'Bluehost';
-	}
 
-	$menu_name = str_replace( '_', ' ', $menu_name );
+	$menu_position = -10;
+	$menu_name = 'Bluehost';
```
Comment (0aveRyan) [inline inc/menu.php]: We can probably nix this variable and just replace within `add_menu_page()`

Comment (wpscholar) [reply inc/menu.php]: Done.

File: bluehost.php (hunk 1 above)
Comment (wpscholar) [reply bluehost.php]: The logged in part is implied with `is_admin` and I don't want to slow down any front-end requests (e.g. membership sites, e-commerce sites, etc. )

**Pull request merged**